src/azure-cli/azure/cli/command_modules/rdbms/custom.py

```diff
@@ -459,4 +474,63 @@ def reject_private_endpoint_connection(cmd, client, resource_group_name, server_
     return _update_private_endpoint_connection_status(
         cmd, client, resource_group_name, server_name, private_endpoint_connection_name, is_approved=False,
         description=description)
+
+
+def server_key_create(client, resource_group_name, server_name, kid):
+
+    """Create Server Key."""
+
```

Comment:
Should the key be in the same resource group with server?

Reply:
The actual Key creation happen in Azure keyvault and that is outside the scope of this commands.  that can be in any subscription/ resource group. Once the key is created in keyvualt the  key will be passed to postgres/mysql server to enable the byok.  The create operation here refers to updating the server with the key created in the key vault and enable encryption.  This is inline with th existing behaviour with the sql server